interactions/client/bot.py

```diff
@@ -1596,9 +1599,17 @@ async def wait_for_component(
         Waits for a component to be interacted with, and returns the resulting context.
 
         .. note::
-            If you are waiting for a select menu, you can find the selected values in ``ctx.data.values``
+            If you are waiting for a select menu, you can find the selected values in ``ctx.data.values``.
+            Another possibility is using the :meth:`self.wait_for_select` method like this:
 
-        :param Union[str, interactions.Button, interactions.SelectMenu, List[Union[str, interactions.Button, interactions.SelectMenu]]] components: The component(s) to wait for
+            .. code-block:: python
+                ctx, values = await bot.wait_for_select(custom_id)
```

Comment:
Why do you have the code example for `wait_for_select` in `wait_for_component`?

Reply:
Because it's an alternative to it and I assume people know wait for component more than wait for select

Reply:
changed anyweays

Reply:
You could say that it is recommended to use `wait_for_select` for selects, and then have the example in the docstring for it. It's counterintuitive to have to reference one function for the usage of another

Reply:
(check unstable)